src/borg/testsuite/archiver/rinfo_cmd.py

```diff
@@ -19,6 +20,7 @@ def test_info(self):
         self.cmd(f"--repo={self.repository_location}", "create", "test", "input")
         info_repo = self.cmd(f"--repo={self.repository_location}", "rinfo")
         assert "Original size:" in info_repo
+        assert "Storage quota:" not in info_repo
```

Comment:
maybe do not address the quota stuff within this test, but rather add a similar one as the other, just without a set quota.